apps/judicial-system/web/pages/verjandi/akaera/[id].ts

```diff
@@ -0,0 +1,3 @@
+import Overview from '@island.is/judicial-system-web/src/routes/Court/components/Overview/Overview'
```

Comment:
Should this not be `routes/Defender/CaseOverview`?

Reply:
No, we are using the Overview screen for defenders in indictments. The Defender/CaseOverview screen is a completely different design. See 👇 

<img width="829" alt="Screenshot 2022-09-27 at 11 12 31" src="https://user-images.githubusercontent.com/3789875/192510875-20819a30-25ea-476f-a8dd-0a2e7b3fe16c.png">

Reply:
Should we when Create a new version for `Defenders` or move the `Court/Overview` to shared?

Reply:
Yea! I've now moved Court/Overview to shared. How do you like this setup?

Reply:
Looks good